dotnetv3/Bedrock-runtime/README.md

```diff
@@ -29,21 +29,6 @@ For prerequisites, see the [README](../README.md#Prerequisites) in the `dotnetv3
 <!--custom.prerequisites.start-->
 <!--custom.prerequisites.end-->
 
-### Single actions
-
-Code excerpts that show you how to call individual service functions.
```

Comment:
Did changing the category to "models" make all of the Bedrock single actions disappear from the READMEs? I notice it in other languages too.

Reply:
I updated WRITEME and regenerated this README so these now appear under the custom category heading.